notebooks/tuning.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<p>I miss some general motivation and intorduction into the topic. A short motivation would allow the listeners to refocus on the new topic and take a deep breath before the next topic starts.</p><p> </p><p>It could include a short wrap-up of the previous parts and an exemplary motivation on grid search and pipelines.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sktime/sktime-tutorial-pydata-global-2022/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>